main/androidhilt/src/mill/main/android/hilt/AndroidHiltTransformAsm.scala

```diff
@@ -0,0 +1,96 @@
+package mill.main.android.hilt
+
+import com.android.build.api.instrumentation.{ClassContext, ClassData}
+import dagger.hilt.android.plugin.transform.AndroidEntryPointClassVisitor
+import mill._
+import org.objectweb.asm._
+
+object AndroidHiltTransformAsm {
+
+  def main(args: Array[String]): Unit = {
+
+    val scanDirectory = os.Path(args.head)
+
+    val destination = os.Path(args.last)
+
+    transformAsm(scanDirectory, os.walk(scanDirectory).filter(_.ext == "class"), destination)
+
+  }
```

Comment:
This is a uncommon entry-point for a Mill "worker". It this because it can run stand-alone (without any Mill infrastructure)? Please add some Scaladoc and include what `args` are supported/expected.